pkg/ccl/crosscluster/settings.go

```diff
@@ -109,3 +109,20 @@ var ReplicateSpanConfigsEnabled = settings.RegisterBoolSetting(
 		"destination system tenant",
 	true,
 )
+
+var LogicalReplanThreshold = settings.RegisterFloatSetting(
+	settings.SystemOnly,
```

Comment:
ApplicationLevel

Reply:
Why Application over System? Is this something we want users tinkering around with?

Reply:
It has more to do with what part of the system the setting is controlling. Typically anything up at the SQL level should be ApplicationLevel whereas anything down in KV is often System level. One way to think about it is whether or not we would want a user in Serverless to be able to modify the setting.

In PCR a lot of the settings are SystemLevel only because PCR is system-tenant-only.  Here, I think most of our settings should probably be ApplicationLevel.